docs/content/commands/npm-team.md

```diff
@@ -24,31 +22,76 @@ Used to manage teams in organizations, and change team memberships. Does not
 handle permissions for packages.
 
 Teams must always be fully qualified with the organization/scope they belong to
-when operating on them, separated by a colon (`:`). That is, if you have a `wombats` team in a `wisdom` organization, you must always refer to that team as `wisdom:wombats` in these commands.
+when operating on them, separated by a colon (`:`). That is, if you have a
+`newteam` team in an `org` organization, you must always refer to that team
+as `org:newteam` in these commands.
 
-If you have two-factor authentication enabled in `auth-and-writes` mode, then you can provide a code from your authenticator with `[--otp <otpcode>]`. If you don't include this then you will be prompted.
+If you have two-factor authentication enabled in `auth-and-writes` mode, then
+you can provide a code from your authenticator with `[--otp <otpcode>]`.
+If you don't include this then you will be prompted.
 
 * create / destroy:
-  Create a new team, or destroy an existing one. Note: You cannot remove the `developers` team, <a href="https://docs.npmjs.com/about-developers-team" target="_blank">learn more.</a>
-* add / rm:
+  Create a new team, or destroy an existing one. Note: You cannot remove the
+  `developers` team, <a href="https://docs.npmjs.com/about-developers-team" target="_blank">learn more.</a>
+
+  Here's how to create a new team `newteam` under the `org` org:
+
+  ```bash
+  npm team create @org:newteam
+  ```
+
+  You should see a confirming message such as: `+@org:newteam` once the new
+  team has been created.
+
+* add:
   Add a user to an existing team, or remove a user from a team they belong to.
```

Comment:
this should probably not mention removing a user, since there's a block for `rm` below